07-alerting-and-notification/README.md

```diff
@@ -157,4 +159,26 @@ Steps to follow:
 **Note:**
 Clicking on the notification name in `Slack` will open a web browser to an unreachable web page with the internal Kubernetes DNS of the `Alertmanager` pod. This is expected. For more information you can check out this [article](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/).
 
+## Debugging a Firing Alert
+
+When an alert fires and sends a notification in Slack it's important that you can debug the problem and find the root cause in a timely manner.
+To do this you can make use of `Grafana` which has already been installed in [Chapter 4 - 04-setup-prometheus-stack](../04-setup-prometheus-stack/README.md) and of `Loki` [Chapter 5- 05-setup-loki-stack](../05-setup-loki-stack/README.md).
+
+Steps to follow:
+
+1. Create a port forward for `Grafana` on port `3000`:
+
+    ```shell
+    kubectl --namespace monitoring port-forward svc/kube-prom-stack-grafana 3000:80
+    ```
+
+2. Open a web browser on [localhost:3000](http://localhost:3000) and log in using the default credentials (`admin/prom-operator`).
+3. Navigate to the [Alerting](http://localhost:3000/alerting) section
+4. From the `State` filter click on the `Firing` option, identify the `emojivoto-instance-down` alert defined in the [Creating a New Alert](#creating-a-new-alert) section and expand it. You should see the following:
+![Grafana Alert](assets/images/grafana-alert.png)
+5. Click on the `See graph` button. From the next page you can observe the count for the number of pods in the `emojivoto` namespace displayed as a metric.
```

Comment:
Is this is metrics graph for the specific resource at that point in time?

Reply:
The user will see what the grafana time range is set to. This is defaulted to "Last 1 hour". We can also instruct the user to update that to his current needs. Those could be 1 hour/12 hours/5 minutes. Depends.